docs/en/api/wrapper/visible.md

```diff
@@ -0,0 +1,23 @@
+# visible()
+
+Assert `Wrapper` or `WrapperArray` is visible.
+
+Returns false if an ancestor element has `display: none` or `visibility: hidden` style.
+
+This can be used to assert that a component is hidden by `v-show`.
+
+- **Returns:** `{boolean}`
+
+- **Example:**
+
+```js
+import { mount } from 'vue-test-utils'
+import { expect } from 'chai'
+import Foo from './Foo.vue'
+
+const wrapper = mount(Foo)
+expect(wrapper.visible()).toBe(true)
+expect(wrapper.find('.is-not-visible').visible()).toBe(false)
+expect(wrapper.findAll('div').visible()).toBe(true)
```

Comment:
Can you remove the findAll examples please

Reply:
Done :)